code/modules/clothing/head/grod_hats.dm

```diff
@@ -0,0 +1,175 @@
+/obj/item/clothing/head/grodhat
+	name = "grod tophat"
+	desc = "A tophat worn by the distinctive grod people."
+	icon_state = "grey"
+	item_state = "grey"
+	icon = 'icons/mob/species/grod/onmob_grod_head.dmi'
+	flags_inv = HIDEHAIR
+	species_restricted = FLAG_HUMAN | FLAG_FELINID | FLAG_FLY | FLAG_IPC | FLAG_LIZARD | FLAG_MOTH | FLAG_OOZELING | FLAG_PLASMAMAN
```

Comment:
what's the reason other species can't wear them?

Reply:
I didn't code this section, but I assume it's related to the sprite possibly looking wonky.